src/bot.js

```diff
@@ -120,7 +122,7 @@ class Bot {
     payload.components[componentName] =
       this.getFalconComponent(
         true,
-        `eu.gcr.io/deployment-pipeline/xc-r2d2-${skin}`,
+        `registry.k8s.xcaliber.io/platform/xcaf/${skin}`,
```

Comment:
doesn't need the tag?

Reply:
no, this registry is completely managed by us

Reply:
aight, thought the `:latest` issue was with the latest version of Docker Compose :D

Reply:
Ah, you mean this tag, it will be added by falcon webpanel using `branch` field